tools/postgres/diff_schema.js

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env node
+/* eslint-disable import/no-extraneous-dependencies */
+const program = require("commander");
+const randomstring = require("randomstring");
+const execSync = require("child_process").execSync;
+const path = require("path");
+const fs = require("fs");
+const glob = require("glob");
+const tmp = require("tmp");
+const rimraf = require("rimraf");
+const replace = require("replace-in-file");
+
+let POSTGRES_URL = process.env.POSTGRES_URL;
+const ORIGINAL_POSTGRES_URL =
+  typeof POSTGRES_URL !== "undefined" ? POSTGRES_URL : "jdbc:postgresql://localhost/webknossos";
+const scriptdir = __dirname;
+const scriptName = __filename;
+let p1;
+let p2;
+let dir1;
+let dir2;
+
+function dump(parameter) {
+  if (parameter === "DB") {
+    POSTGRES_URL = ORIGINAL_POSTGRES_URL; // this environment variable is passed to dump_schema.sh
+    return dumpToFolder();
+  } else {
+    const { dbName, dbHost } = initTmpDB();
+    console.log(`Creating DB ${dbName}`);
+    execSync(`psql -U postgres -h ${dbHost} -c "CREATE DATABASE ${dbName};"`, {
+      env: { PGPASSWORD: "postgres" },
+    });
+    try {
+      loadDataIntoDB(parameter, dbHost, dbName);
+      return dumpToFolder();
+    } catch (err) {
+      console.log(err);
+      process.exit(1);
+    } finally {
+      console.log(`CLEANUP: DROP DATABASE ${dbName}`);
+      execSync(`psql -U postgres -h ${dbHost} -c "DROP DATABASE ${dbName};"`, {
+        env: { PGPASSWORD: "postgres" },
+      });
+    }
+  }
+  return null;
+}
+
+function initTmpDB() {
+  const tempDbName = generateRandomName();
+  const postgresDirname = path.dirname(ORIGINAL_POSTGRES_URL);
+  POSTGRES_URL = `${postgresDirname}/${tempDbName}`; // this environment variable is passed to dump_schema.sh
```

Comment:
overwriting the global variable doesn't seem to be very elegant. Can you use a regular local variable?